sunpy/timeseries/sources/eve.py

```diff
@@ -74,22 +76,25 @@ def plot(self, axes=None, **kwargs):
             The plot axes.
         """
         self._validate_data_for_plotting()
-        names = ("Flux \n 0.1-7nm", "Flux \n 18nm", "Flux \n 26nm", "Flux \n 30nm", "Flux \n 36nm")
+        column_names = {"QD": "Flux \n 0.1-7nm", "CH_18": "Flux \n 18nm",
+                        "CH_26": "Flux \n 26nm", "CH_30": "Flux \n 30nm", "CH_36": "Flux \n 36nm"}
         colors = ('tab:blue', 'tab:orange', 'tab:green', 'tab:red', 'tab:purple',
                   'tab:brown', 'tab:pink', 'tab:gray', 'tab:olive', 'tab:cyan')
         predefined_axes = False
+        if column is None:
+            column = self.to_dataframe().columns
         if isinstance(axes, np.ndarray):
             predefined_axes = True
         elif axes is None:
-            axes = self.to_dataframe().plot(subplots=True, sharex=True, **kwargs)
-        for i, ax in enumerate(axes):
+            axes = self.to_dataframe()[column].plot(subplots=True, sharex=True, **kwargs)
+        for i, name in enumerate(self.to_dataframe()[column]):
```

Comment:
is `name` used somewhere?

Reply:
I thought of using it to set the label but I was uncertain about it.
Okk I will make the changes accordingly